torch/csrc/jit/codegen/cuda/test/test_gpu3.cpp

```diff
@@ -6588,6 +6588,28 @@ TEST_F(NVFuserTest, FusionVectorizeRepro1843_CUDA) {
   testValidate(fusion, cg_outputs, {t1, t0}, {ref}, __LINE__, __FILE__);
 }
 
+TEST_F(NVFuserTest, FusionPersistentReduction_CUDA) {
```

Comment:
This test is unrelated to the change in this PR, I just feel that it is a good example to demonstrate normalization + reduction.